images/tensorflow/Dockerfile

```diff
@@ -21,9 +21,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends git
 
 RUN pip3 install cloud-tpu-client pyyaml
 
+ARG tensorflow_text_version=latest
+ARG keras_version=latest
+
 # HACKs: Remove this once they are added to the Model Garden requirements
 RUN pip3 install tensorflow-recommenders --no-deps
-RUN pip3 install tensorflow-text==2.6.0rc0 --no-deps
+RUN pip3 install tensorflow-text==${tensorflow_text_version} --no-deps
+RUN pip3 install keras==${keras_version} --no-deps
```

Comment:
Is this necessary? And will it work for nightly? New nightly images have keras-nightly installed already

Reply:
Removed it. 
Actual error is that while installing requirements.txt packages, it is indirectly installing TF-2.6.0. Now it has been fixed.